### ISSUE TITLE
Version 0.1/1.0 what is needed?

### ISSUE THREAD

**Issue by NogginBops** [SpaceKrakens/SKON.NET#10]
What needs to be implemented before a v0.1 or v1.0 can be released? How should we handle future versions? and how should we handle different SKON versions?

**Comment by Tranberry** [comment]
A working example for something that looks like a "real world" project. If there's a somewhat stable project that can easily be ported that would be a great proof of usability.

**Comment by NogginBops** [comment]
Sounds like a good idea. Do you have anything in mind or like a general idea for such a project?

**Comment by Tranberry** [comment]
No, I don't :( @SpaceKrakens/superkraken do you have any ideas?
